src/IO/H5/Dat.hpp

```diff
@@ -0,0 +1,110 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class h5::Dat
+
+#pragma once
+
+#include <array>
+#include <iosfwd>
+#include <vector>
+
+#include "IO/H5/Object.hpp"
+#include "IO/H5/OpenGroup.hpp"
+
+namespace h5 {
+/*!
+ * \ingroup HDF5
+ * \brief Represents a multicolumn dat file inside an HDF5 file
+ *
+ * Stores data that is normally stored inside a dat file but inside a dataset
```

Comment:
Stores (what it stores) instead of referring to SpEC stuff that future users will be completely ignorant of